src/scss/04-patterns/03-interaction/_input-with-icon.scss

```diff
@@ -69,6 +69,8 @@
 		&.validation-message {
 			display: block;
 			position: absolute;
+			// the !important is used to override a more specific style containing ThemeGrid_Width context which is not needed here
+			left: 0 !important;
```

Comment:
Then let's make this rule more specific, is still better than !important

Reply:
Mixed feelings about it... but's done... ;)